.github/workflows/ohos.yml

```diff
@@ -243,6 +243,7 @@ jobs:
         run: hitrace-bench --bencher -b "org.servo.servo" -p "https://www.servo.org" -n 5
       - name: Getting bencher
         uses: bencherdev/bencher@main
-      - name: Uploading to bencher.dev
+      - name: Getting model name and Uploading to bencher.dev
         run: |
-          bencher run --adapter json --file bench.json --project '${{ env.BENCHER_PROJECT }}' --token '${{ secrets.BENCHER_API_TOKEN }}' --github-actions '${{ secrets.GITHUB_TOKEN }}'
+          phone_name=$(hdc bugreport | head -n 20 | grep MarketName | awk '{for (i=2; i<NF; i++) printf $i " "; print $NF}' -)
```

Comment:
I would put this in a seperate step. 
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable explains how you can save the name into an environment variable, which you could then read in the bench step.